wasmedge-ggml/README.md

```diff
@@ -0,0 +1,268 @@
+# Llama Example For WASI-NN with GGML Backend
+
+[See it in action!](https://x.com/juntao/status/1705588244602114303)
+
+## Requirement
+
+### For macOS (apple silicon)
```

Comment:
Let's reduce the situation:

1. macOS (apple silicon & intel chip): Both are supported. However, the Metal(GPU support) is only available on Apple silicon.
2. Linux (with CUDA enabled)
3. Linux (without CUDA enabled)

Reply:
Since the installer will detect the Metal/CUDA, we no longer need to separate the details.

Reply:
README updated.